modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -51,15 +75,69 @@ See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for addit
                 next-hop-interface: <NIC1_name>
 ----
 <1> Add NMState YAML syntax to configure host interfaces.
-
-[TIP]
-====
-Consider saving the `networkConfig` YAML syntax to a file and testing it using the NMState command line interface before including it in the `install-config.yaml` file, because the installer will not check the NMState YAML syntax. Execute `nmstatectl gc <yaml-config>` to test the syntax. Errors in the YAML syntax might result in a failure to apply the network configuration. Additionally, maintaining the validated YAML syntax is useful when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
-====
-
-The most common use case for this functionality is to specify a static IP address on the `baremetal` network, but you can also configure other networks such as a storage network. This functionality will also support other NMState features such as VLAN, VXLAN, bridges, bonds, routes, MTU, and DNS resolver settings.
-
++
 [IMPORTANT]
 ====
 Once deployed, you cannot modify the `networkConfig` configuration setting of `install-config.yaml` file to make changes to the host network interface. Use the Kubernetes NMState Operator to make changes to the host network interface after deployment.
 ====
++
+In the case of deploying {product-title} without a DHCP server on the baremetal network, use the following procedure to configure a static IP address for the bootstrap VM using Ignition:
+
+. Create `bootstrap_config.sh` file:
++
+.Example
+[source,terminal]
+----
+$ cat bootstrap_config.sh
+#!/bin/bash
+BOOTSTRAP_CONFIG="[connection]
+type=ethernet
+interface-name=ens3
+[ethernet]
+[ipv4]
+method=manual
+addresses=192.168.123.101/24
+gateway=192.168.123.1
+dns=192.168.123.1"
+cat > bootstrap_network_config.ign << EOF
+{
+  "path": "/etc/NetworkManager/system-connections/ens3.nmconnection",
+  "mode": 384,
+  "contents": {
+    "source": "data:text/plain;charset=utf-8;base64,$(echo "${BOOTSTRAP_CONFIG}" | base64 -w 0)"
+  }
+}
+EOF
+----
++
+. Run `bootstrap_config.sh` to create the `json` configuration file:
++
+.Example
+[source,terminal]
+----
+$ ./bootstrap_config.sh
+----
++
+. Insert the json configuration file into the Ignition file:
```

Comment:
"Insert the json config" and "Inject the ignition file" are basically synonymous and both occur in the jq command in line 134.
The 'mv' command in line 126 is a backup of the original bootstrap file and is an optional step.

Reply:
Updated as suggested.